tequilapi/endpoints/identities.go

```diff
@@ -0,0 +1,32 @@
+package endpoints
+
+import (
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+	"github.com/mysterium/node/identity"
+	"github.com/mysterium/node/service_discovery/dto"
+	"github.com/mysterium/node/tequilapi/utils"
+)
+
+type IdentitiesApi struct {
+	getIdentityArray func() []dto.Identity
+}
+
+func IdentityHandlers(idm *identity.IdentityManager) *IdentitiesApi {
+	return &IdentitiesApi{getIdentityArray: idm.GetIdentities}
+}
```

Comment:
Why do we need intermediate getIdentityArray func if we depend on IdentityManager anyway? just store the reference in the struct and call GetIdentities when needed.
Another way would be get rid of IdentityManager dependecy and pass getIdentityArray to IdentityHandlers. But current situation is overabstracted and doesnt add anything

Reply:
fixed